kinbot/mess.py

```diff
@@ -653,21 +728,24 @@ def run(self):
         while uq_iter < self.par['uq_n']:
             self.write_submitscript(submitscript, uq_iter)
             while len(pids) > self.par['uq_max_runs']:
-                time.sleep(5)
+                time.sleep(1)
                 for pid in pids:
                     stat = self.check_running(pid)
-                    if stat == 0:
+                    if stat == 1:  # rosalind
+                    # if stat == 0: #  menten
                         pids.remove(pid)
                         pid_stats.append(stat)
-            
             pid = self.submit(submitscript)
             pids.append(pid)
 
             if self.par['uq_n'] < self.par['uq_max_runs']:
-                stat = 1
-                while stat != 0:
+                # stat = 1  # menten
+                stat = 0  #rosalind
+                # while stat != 0:  # menten
+                while stat != 1:  # rosalind
```

Comment:
Double check these values on menten/blodgett. At one point you needed to swap the value when using slurm vs pbs.